pages/1.12/release-notes/1.12.4/index.md

```diff
@@ -0,0 +1,244 @@
+
+    For more information about using Docker images with DC/OS Universal Container Runtime (UCR), see [Universal Container Runtime](/1.13/deploying-services/containerizers/ucr/).
+
+- Changes the behavior for tasks when there are failed resource providers (ASF-2839).
+
+    With this release, the resource manager will no longer ask all resource providers to publish all allocated resources. With this change, a failed resource provider would only fail tasks that wanted to use the failed resource provider's resources.
+
+## Metrics
+- Corrects the service endpoint values and service address-based statistics that are returned when the `statsd` metrics input plugin is enabled (COPS-3279, COPS-3576, DCOS-37703, DCOS-39703).<!--Also in previous RN, 1.12.4-->
+
+- Adds HTTP request time and HTTP request size metrics for Metronome-scheduled jobs (DCOS_OSS-5020). 
+
+- Enables framework names to be properly decoded in metric tags (DCOS_OSS-5039).
+
+    Mesos masters allow spaces in framework names by using percent-encoding (%20) of the framework name. This release updates the Telegraf plugin to enable it to decode the framework name and export metrics with the correct tags.
+
+- Fixes issues that caused some DC/OS components to crash when the `/tmp` directory is mounted using the `noexec` option (DCOS-53077).
+
+- Modifies the Telegraf configuration settings to improve how metrics are collected on busy agents (DCOS-50994).
+
+## Networking
+- Adds a timeout to the Mesos network overlay module to prevent the overlay master from getting stuck in RECOVERING mode (COPS-4167, COPS-4747, DCOS_OSS-4575, DCOS-47930).
+
+- Includes a fix that prevents duplicate virtual IP addresses (VIP) from showing up after the manual removal of an agent or framework initiated by operator intervention (COPS-4703, DCOS_OSS-4963, DCOS-51157).
+
+    Previously, if an operator manual removed an agent or framework while performing routine maintenance or by issuing the framework TEARDOWN call, the Mesos agent might return information about terminated tasks in a response field normally reserved for running tasks. This incorrect response in turn resulted in the `dcos-net` networking component adding a duplicate VIP entry. The fix in this release prevents the `dcos-net` networking component from adding a duplicate virtual IP address when an agent or framework is removed.
+
+- Resolves a deadlock or race condition that could prevent one or more nodes in a cluster from generating a routing table that forwards network traffic through Marathon load balancing properly (COPS-3585). 
```

Comment:
That issue has been fixed in DC/OS 1.12.0, but you are constantly adding it to fixed bugs or known issues in every 1.12.x release notes. why? 😞

Reply:
Not my preference. I have brought up this question before--but I think it has to do with release tracking in JIRA. In the next release, we might be able to eliminate the duplication by only publishing change log items disconnected from the JIRA release dashboard.

Reply:
@lsgunnlsgunn I understand this list matches issues tagged with 1.12.x and I can actually see value in including _all_ patches since the initial major release but would argue that we should include the actual fix version as one might read this as _fixed in 1.12.4+ equals broken in 1.12.3-_ whish is not correct and leads to confusion. What do you think?

Reply:
Wait, the notes do not include _all_ patches since the initial major release but are limited to the ones fixed in 1.12.4 which makes me wonder why this was included again. Is there something wrong with how we tagged it? And can you please remove the note to not confuse our customers?

Reply:
I think your point is well taken--though I've gotten conflicting input. If you can explicitly indicate what you would like me to remove, I'll open a new branch and PR to address that, but I'm not sure what note you are referring to. Thanks for your help.

Reply:
`COPS-3585` was fixed with `DCOS-38600` in Oct last year and shipped with 1.12.0 thus shouldn't be included in any of the release notes except for the 1.12.0 one, thus we should remove the resp. note. 

P.S. Looked into `COPS-3585` and it seems that it was tagged with `DC/OS 1.12` instead of `DC/OS 1.12.0` which is probably why it pops up again and again. I've fixed the COPS issue. I'm sorry for the noise it created. 

<img src="https://media.giphy.com/media/fV8iuSEwLQ6005diSh/giphy.gif" />